app/views/partials/view-claim/claim.html

```diff
@@ -54,7 +54,9 @@
           <select id="claim-expense-{{ expense.ClaimExpenseId }}-status" data-id="{{ expense.ClaimExpenseId }}" name="claim-expense-{{ expense.ClaimExpenseId }}-status"
             class="form-control action-select claim-expense-status">
             <option {% if expense['Status'] == undefined %} selected {% endif %}>Select</option>
+            {% if expense['ExpenseType'] != 'car' and expense['ExpenseType'] != 'train' %}
```

Comment:
train expenses will have a cost if not an advance claim, use the ClaimExpense.Cost value which should be zero if it needs APPROVED_DIFF_AMOUNT